package-lock.json

```diff
@@ -13877,7 +13877,7 @@
         "hook-std": "^2.0.0",
         "hosted-git-info": "^3.0.0",
         "lodash": "^4.17.15",
-        "marked": "^1.0.0",
+        "marked": "^2.0.0",
```

Comment:
Should this be `2.0.1`?

Reply:
@jasonodonnell I Don't think so. It will update you to the most recent major version (the first number). ^2.0.0 will match any 2.x.xrelease including 1.2.0, but will hold off on 3.0.0 . when I run: `npm i --package-lock-only` it still gives me the same thing

Reply:
@jasonodonnell I Don't think so. It will update you to the most recent major version (the first number). ^2.0.0 will match any 2.x.xrelease including 1.2.0, but will hold off on 3.0.0 . when I run: npm i --package-lock-only it still gives me the same thing